source/docs/ROS/getting-started.rst

```diff
@@ -0,0 +1,67 @@
+Getting Started
+===============
+
+Downloading the Base Repo
+-------------------------
+
+To get started make sure ROS is installed on your machine:
+
+.. note:: The following instructions can be found in the Studica ROS repository on github: ``https://github.com/studica/ros-dev``.
+
+Installing ROS
+^^^^^^^^^^^^^^
+
+1. If you haven't already, run ``installNoetic.sh`` to install ROS Noetic. For more information, visit http://wiki.ros.org/noetic/Installation/Debian
+
+2. Download catkin tools from https://catkin-tools.readthedocs.io/en/latest/installing.html, then run ``sudo apt install python3-catkin-tools python3-osrf-pycommon`` to finalize the install.
+
+.. attention:: Running the ``installNoetic.sh`` script will require some time, this process can take approximately 3 hours. 
+
```

Comment:
We have a ROS image that they will be downloading. This will save time and ensure dependencies are correct. Also, we will be pushing the dev repo to a different repo for public release.

Reply:
So the catkin tools are going to be included in the ROS image? Should I take this part out then if the ROS image is ready?

Reply:
Do you know the release repo so I can reference it in the documentation

Reply:
Yes, the catkin tools will be preinstalled. I do not know the link yet as we will need to clean up the base project before release.

Reply:
ok